CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+Not released yet
+----------------
+
+- Update where `ConnectionDoesNotExist` is imported from
```

Comment:
Changes to the test-suite don't need appearance in the changelog

Reply:
I've removed the changelog entry and rebased on current `master`.